cba_settings_userconfig/$PREFIX$

```diff
@@ -0,0 +1 @@
+cba_settings_userconfig
```

Comment:
Why is this needed?

Reply:
I think this may be an unintended copy-paste error on my part when I was probably searching for something.